src/ScheduleFunctions.cpp

```diff
@@ -1211,17 +1218,27 @@ class InjectFunctionRealization : public IRMutator {
             }
 
             string parent_prefix = fuse_level.func() + ".s" + std::to_string(fuse_level.stage_index()) + ".";
+            int parent_var_index = (i - start_fuse) + (int)parent_dims.size() - 1 - fused_vars_num;
```

Comment:
Ditto underflow question. Is it overkill to add `internal_assert(parent_var_index >=0)`?

Reply:
Good point, I've added assert.